### DISCUSSION TITLE
Packaging applications with pre-release info in the version

### DISCUSSION THREAD

**Discussion by jcasale** [wixtoolset/issues#7717] [Questions] [answered]
I have to start packaging an application that follows the _major.minor_ pattern with release candidates, for example:

- 1.0
- 1.1rc1
- 1.1rc2
- 1.1

Is there any other approach other than to make all releases of v1.1 the same version and set `MajorUpgrade/AllowSameVersionUpgrades=Yes` and differentiate them in the title/description?

**Comment by robmen** [comment, marked as answer]
You get three parts to the MSI version. Use a monotonically increasing value for the third part. Semver is cute but, in the end, just make sure later releases of `v#.#.#` are bigger than older versions of `v#.#.#`.
